calendar-api.js

```diff
@@ -91,6 +91,119 @@ module.exports = {
+   * @param [parameters.user.email] {string} The SMTP address of the user. If absent, the `/Me` segment is used in the API URL.
+   * @param [parameters.user.timezone] {string} The timezone of the user.
+   * @param [parameters.calendarId] {string} The calendar id. If absent, the API calls the `/User/calendarview` endpoint. Valid values of this parameter are:
+   * 
+   * - The `Id` property of a `Calendar` entity
+   * - `Primary`, the primary calendar is used. It is used by default if no Id is specified
+   * 
+   * @param [callback] {function} A callback function that is called when the function completes. It should have the signature `function (error, result)`.
+   * 
+   * @example var outlook = require('node-outlook');
+   * 
+   * // Set the API endpoint to use the 2.0 version of the api
+   * outlook.base.setApiEndpoint('https://outlook.office.com/api/v2.0');
+   * 
+   * // This is the oAuth token 
+   * var token = 'eyJ0eXAiOiJKV1Q...';
+   * 
+   * // Pass the user's email address
+   * var userInfo = {
+   *   email: 'sarad@contoso.com'
+   * };
+   *
+   * // You have to specify a time window
+   * var startDateTime = "2017-01-01";
+   * var endDateTime = "2017-12-31";
+   * 
+   * var apiOptions = {
+   *   token: token,
+   *   calendarId: 'Primary', // Not mandatory, the Primary calendar will be used by default, but you can specify another calendar Id
+   *   user: userinfo,
+   *   startDateTime: startDateTime,
+   *   endDateTime: endDateTime
+   * };
+   *
+   * outlook.mail.syncEvents(apiOptions, function(error, events) {
+   *   if (error) {
+   *     console.log('syncEvents returned an error:', error);
+   *   } else {
+   *     // Do something with the events.value array
+   *     // Then get the @odata.deltaLink
+   *     var delta = messages['@odata.deltaLink'];
+   * 
+   *     // Handle deltaLink value appropriately:
+   *     // In general, if the deltaLink has a $skiptoken, that means there are more
+   *     // "pages" in the sync results, you should call syncEvents again, passing
+   *     // the $skiptoken value in the apiOptions.skipToken. If on the other hand,
+   *     // the deltaLink has a $deltatoken, that means the sync is complete, and you should
+   *     // store the $deltatoken value for future syncs.
+   *     //
+   *     // The one exception to this rule is on the intial sync (when you call with no skip or delta tokens).
+   *     // In this case you always get a $deltatoken back, even if there are more results. In this case, you should
+   *     // immediately call syncMessages again, passing the $deltatoken value in apiOptions.deltaToken.
+   *   }
+   * }
+   */
+  
+  syncEvents: function(parameters, callback) {
+      var userSpec = utilities.getUserSegment(parameters);
+      var calendarSpec = parameters.calendarId === undefined ? '' : "/calendars('" + parameters.calendarId + "')"
```

Comment:
That's it, and also, it shouldn't be /calendars(calendar_id), but /calendars/calendar_id, like this:
`var calendarSpec = parameters.calendarId === undefined ? '' : "/calendars/" + parameters.calendarId;`